pasta_eln/GUI/project.py

```diff
@@ -99,12 +100,10 @@ def projHeader(self) -> None:
     comment.setStyleSheet(f"QTextEdit {{ border: none; padding: 0px; background-color: {bgColor}; "\
                           f"color: {fgColor} }}")
     comment.setReadOnly(True)
-    comment.document().setTextWidth(commentW.width())
```

Comment:
Not sure, if this is the right approach to do it, normally the comment box should resize automatically as per the content, the idea of using scroll area or text area is to do such things on it's own, why don't we take such an approach in general for all the text boxes or areas?